models/classes/Platform/Repository/Lti1p3RegistrationRepository.php

```diff
@@ -133,12 +115,38 @@ private function getDefaultPlatform(): Platform
             self::PLATFORM_ID,
             self::PLATFORM_ID,
             rtrim(ROOT_URL, '/'),
-            self::OIDC_URL
+            self::OIDC_URL,
+            self::OAUTH_URL
         );
     }
 
     private function getLtiProviderService(): LtiProviderService
     {
         return $this->getServiceLocator()->get(LtiProviderService::SERVICE_ID);
     }
+
+    private function createRegistrationByProvider(LtiProvider $ltiProvider): ?Registration
+    {
+        $toolKeyChain = $this->getToolKeyChainRepository()
+                ->findAll(new KeyChainQuery($ltiProvider->getId()))
+                ->getKeyChains()[0] ?? null;
```

Comment:
should we take into consideration that the `->findAll(new KeyChainQuery($ltiProvider->getId()))->getKeyChains()``might return a different key then 0 but still return a response ? 
so my question would be if we should use the `current()` instead of the `[0]`